src/Files.Launcher/Helpers/CloudDrivesDetector.cs

```diff
@@ -58,6 +58,12 @@ private static Task<List<CloudProvider>> DetectGenericCloudDrive()
                         continue;
                     }
 
+                    //Nextcloud specific
+                    if (namespaceKey?.GetValue("ApplicationName") != null)
```

Comment:
How sure are we that only Nextcloud has an "ApplicationName" key? To be sure this could be chaged to: `if (namespaceKey?.GetValue("ApplicationName") =="Nextclout")`

Reply:
could not find any documentation so suggested change have been applied in 1b4fe03